### PR TITLE
prevent mass import

### PR DESCRIPTION
**Pull request by gumaerc**
#### Pre-Flight checklist

- [ ] Testing
  - [x] Code is tested
  - [ ] Changes have been manually tested

#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-studio/issues/1167

#### What's this PR do?
After recently releasing the new OCW site live and going through a month long QA process getting the content into shape, it would be a big problem if `import_ocw_course_sites` were to be run unfiltered or with a vague filter.  The tasks that `import_ocw_course_sites` runs destructively updates content on imported legacy sites, and since a lot of manual work was done in production to get the site ready for launch, we want to avoid destroying this data.  This PR does a few things to prevent that:

 - Either `--filter` or `--filter-json` is now required when running the command
 - A confirmation step was added that lets you know how many sites will be imported, asking you to type `y` and hit enter before proceeding

#### How should this be manually tested?
 - Get `ocw-studio` up and running locally on this branch, making sure you've configured AWS access (okay to match RC) as well as a private Github org for publishing
 - Put the following in a `courses.json` file at the root of your `ocw-studio` folder:
```
[
  "8-591j-systems-biology-fall-2004",
  "15-057-systems-optimization-spring-2003",
  "6-776-high-speed-communication-circuits-spring-2005"
]
```
 - Once up and running, run the following commands:
   - `docker-compose run --rm web ./manage.py import_ocw_course_sites -b ocw-to-hugo-output-qa` - This should throw an error of `CommandError: This command cannot be run unfiltered.  Use the --filter or --filter-json argument to specify courses to import.`
   - `docker-compose run --rm web ./manage.py import_ocw_course_sites -b ocw-to-hugo-output-qa --filter "8-591j-systems-biology-fall-2004, 15-057-systems-optimization-spring-2003, 6-776-high-speed-communication-circuits-spring-2005"` - This should prompt you to confirm importing 3 courses, press y and then enter to confirm
   - `docker-compose run --rm web ./manage.py import_ocw_course_sites -b ocw-to-hugo-output-qa --filter-json courses.json` - This should also prompt you to confirm importing 3 courses, press y and then enter to confirm
   - `docker-compose run --rm web ./manage.py import_ocw_course_sites -b ocw-to-hugo-output-qa --filter "8-591j-systems-biology-fall-2004, 15-057-systems-optimization-spring-2003, 6-776-high-speed-communication-circuits-spring-2005"` - This should prompt you to confirm importing 3 courses, type anything other than `y` and hit enter and you should see `Aborting...`

### CODE REVIEW

File: ocw_import/management/commands/import_ocw_course_sites.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 """ Import OCW course sites and content via ocw2hugo output """
+import json
 import pydoc
 
 from django.conf import settings
-from django.core.management import BaseCommand
+from django.core.management import BaseCommand, CommandError
 from mitol.common.utils.datetime import now_in_utc
+from six.moves import input
```
Comment (ChristopherChudzicki) [inline ocw_import/management/commands/import_ocw_course_sites.py:8]: **Question:** Do we need six? We're not trying to support Python 2...

**Review by ChristopherChudzicki — COMMENTED** [review]
Still running commands. Two questions

1. One about six... see comment
2. Maybe not worth the effort right now, and not really the point of this PR... but question: could we do more of the filtering on s3-server-side with with prefix? Right now we fetch all the paths then filter on our end, right?

**Review by ChristopherChudzicki — APPROVED** [review]
👍 ...guard rails 🎉.

still questions, but they not blocking.

File: ocw_import/management/commands/import_ocw_course_sites.py (hunk 1 above)
Comment (gumaerc) [reply ocw_import/management/commands/import_ocw_course_sites.py]: 1.  I was confused about how `input` works in management commands, and it seems you don't need to use the `six` compatibility layer to get that to work.  I switched it to just use the native `input` function.

2.  The server side filtering is definitely a massive time sink and we should bake filtering into the actual use of the S3 API, but that's out of scope for this PR.  I made issues for that and another thing I noticed that we should change now that editing is turned off in Plone here:

https://github.com/mitodl/ocw-studio/issues/1217
https://github.com/mitodl/ocw-studio/issues/1218

**Pull request merged**